app/javascript/modules/listingDetails/ListingDetailsFeatures.tsx

```diff
@@ -73,6 +82,8 @@ export const ListingDetailsFeatures = ({ listing, imageSrc }: ListingDetailsFeat
     depositSubtext.push(t("listings.features.mayBeHigherForLowerCreditScores"))
   }
 
+  const translations = listing.translations
+
   return (
     <ListingDetailItem
```

Comment:
If the FeatureItem has a prop `toTranslate` to false, I didn't pass in the translations.

Reply:
Makes sense!